packages/cli/src/external-hooks.ts

```diff
@@ -94,7 +96,14 @@ export class ExternalHooks {
 		};
 
 		for (const externalHookFunction of this.externalHooks[hookName]) {
-			await externalHookFunction.apply(externalHookFunctions, hookParameters);
+			try {
+				await externalHookFunction.apply(externalHookFunctions, hookParameters);
+			} catch (cause) {
+				// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+				const error = new ApplicationError(`External hook "${hookName}" failed`, { cause });
```

Comment:
1. We could use `ensureError` to prevent the lint exception.
2. We could add `'external-hooks'` to `tags` for filtering.

Reply:
1. Does this linting rule even provide any value here? `cause` being `unknown` is perfectly acceptable here.
2. We can use the error message for filtering

Reply:
1. True, mostly noise.
2. My thinking was that it'd be more useful to have the actual error message in the Sentry event title rather than `External hook "${hookName}" failed` which can group lots of disparate errors, especially as every hook is an array of callbacks.

Reply:
2. I'm hoping that this won't happen frequently enough, and we'll fix external hook as soon as possible. if that happens, we shouldn't need to worry about grouping 🤞🏽